test/contrib/einstein/test_steinvi.py

```diff
@@ -126,7 +126,14 @@ def model(features, labels):
 def test_run_smoke(kernel, problem, method):
```

Comment:
@fehiepsi do I remove the `mark.xfail` decorator?

``` python
@pytest.mark.xfail(
    os.getenv("JAX_CHECK_TRACER_LEAKS") == "1", reason="Expected tracer leak"
)
```

Reply:
I'm going to follow [this](https://github.com/pyro-ppl/numpyro/pull/2001/files#diff-d14f559f375bf09ec589feb7b9f6393838d4d5d767c5501c50c8db5c47da5dfcL523) and remove it.
Running 
```
JAX_CHECK_TRACER_LEAKS=1 pytest -vs test/contrib/einstein/test_steinvi.py
```
passes now.

Reply:
Could you also remove the CI test: https://github.com/pyro-ppl/numpyro/blob/master/.github/workflows/ci.yml#L88 ?